resources/views/gallery/file/_form.blade.php

```diff
@@ -8,6 +8,8 @@
 
                 @if ($image->isFont())
                     <x-helper text="This file is a font file." />
+                @elseif ($image->isFile())
+                    <x-helper text="This file is from an entity asset." />
```

Comment:
This test is incorrect. webp and gifs are both valid in entity assets and the gallery. !hasThumbnail() and the message could be `This file can't be previewed.` instead of two ifs